cli/commands/command/list.go

```diff
@@ -0,0 +1,129 @@
+package command
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"path"
+
+	corev2 "github.com/sensu/sensu-go/api/core/v2"
+	"github.com/sensu/sensu-go/cli"
+	"github.com/sensu/sensu-go/cli/cmdmanager"
+	"github.com/sensu/sensu-go/cli/commands/helpers"
+	"github.com/sensu/sensu-go/cli/elements/table"
+	"github.com/spf13/cobra"
+)
+
+// ListCommand adds command that allows a user to list installed command
+// plugins.
+func ListCommand(cli *cli.SensuCli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "lists installed sensuctl commands",
+		RunE:  listCommandExecute(cli),
+	}
+
+	return cmd
+}
+
+func listCommandExecute(cli *cli.SensuCli) func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		// If args are provided print out usage
+		if len(args) > 0 {
+			_ = cmd.Help()
+			return errors.New("invalid argument(s) received")
+		}
+
+		manager, err := cmdmanager.NewCommandManager(cli)
+		if err != nil {
+			return err
```

Comment:
If no command plugin is installed, we will return an error that's not really UX friendly (e.g. `Error: open /Users/splourde/Library/Caches/sensu/sensuctl/commands.db: no such file or directory`). We should probably try to wrap this error and make it more user friendly (e.g. `no command plugin found`).